<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntficWindow</class>
 <widget class="QMainWindow" name="IntficWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>823</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Intfic Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="intficmain">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1101</width>
      <height>751</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="font">
        <font>
         <family>Noto Mono</family>
        </font>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
       <property name="openLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>760</y>
      <width>1101</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="quitButton">
       <property name="toolTip">
        <string>Close window</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/quit</normaloff>:/icons/quit</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="toolTip">
        <string>Start afresh</string>
       </property>
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boatButton">
       <property name="toolTip">
        <string>Magic Boat trip</string>
       </property>
       <property name="text">
        <string>boat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="thiefButton">
       <property name="toolTip">
        <string>Killing the Thief</string>
       </property>
       <property name="text">
        <string>thief</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mineButton">
       <property name="toolTip">
        <string>Diamond mining</string>
       </property>
       <property name="text">
        <string>mine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="balloonButton">
       <property name="toolTip">
        <string>Balloon AND volcano</string>
       </property>
       <property name="text">
        <string>balloon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="demoButton">
       <property name="toolTip">
        <string>Complete walkthrough, at readable speed</string>
       </property>
       <property name="text">
        <string>demo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
